scootapi/setup/ports.go

```diff
@@ -0,0 +1,25 @@
+package setup
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"time"
+)
+
+// WaitForPort waits ~5 seconds for a process to listen to the port, and returns an error if
+// the port remains open
+func WaitForPort(port string) error {
```

Comment:
this functionality also lives in closed source swarmtest.go, may want to make note to re-use this one later